### PR TITLE
fix app header mobile and networks menu arrows

### LINKED ISSUE #3162: Add close button to mobile menu
**Describe the bug**

Currently a user can only close the mobile menu by clicking on a list item. Which he might not want. So let's add the close button to the mobile menu as well.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: 
 - Browser: 
 - Version: 

**Smartphone (please complete the following information):**
 - Device: 
 - OS: 
 - Browser 
 - Version 

**Additional context**

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #3162

**Description:**
Some fixes for the mobile menu display affecting the AppMenu.vue file.

Fixed the app header not showing when opening the app menu in mobile. 
Also fixed the inconsistency in the app menu with Networks showing when turning into mobile.

Thank you! 🚀

---

For contributor:

- [x] Added changes entries. Run `yarn changelog` for a guided process.
- [x] Reviewed `Files changed` in the github PR explorer
- [x] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
This is the fixed view
image

**Comment by jbibla** [comment]
we need to leave this as is for now. there is no other place where the network link is exposed.

**Comment by Bitcoinera** [comment]
Got it. Should I then make a separate branch for just the app header?

**Comment by faboweb** [comment]
@Bitcoinera just hide the arrow on line 81 instead of the whole link ;)

**Comment by Bitcoinera** [comment]
Done!

File: src/components/common/AppMenu.vue [hunk 1]
```diff
@@ -258,6 +258,12 @@ export default {
   font-weight: 500;
 }
 
+@media screen and (max-width: 667px) {
```
Comment (faboweb) [inline src/components/common/AppMenu.vue]: is there a reason you don't just apply `hide-xs` to line 81?

Comment (Bitcoinera) [reply src/components/common/AppMenu.vue]: Clumsiness? :laughing:

**Pull request merged**